OWML.ModHelper.Menus/ModMenu.cs

```diff
@@ -23,12 +23,12 @@ public class ModMenu : IModMenu
         public List<IModComboInput> ComboInputs { get; private set; }
         public List<IModNumberInput> NumberInputs { get; private set; }
 
-        private readonly IModConsole _console;
-        private LayoutGroup _layoutGroup;
+        protected LayoutGroup LayoutGroup;
+        protected readonly IModConsole Console;
```

Comment:
not really a fan of variable names being the same case/capitalisation as the type - can make it harder to read what's meant to be a type and what's meant to be a variable (and doesn't Console conflict with System.Console?)

Reply:
Well, it's the convention 🤷 
We can choose against conventions but I'd need a good reason. ;)

Reply:
now Layout and MyConsole